delta/service/src/main/scala/ch/epfl/bluebrain/nexus/delta/service/syntax/EventLogSyntax.scala

```diff
@@ -0,0 +1,48 @@
+package ch.epfl.bluebrain.nexus.delta.service.syntax
+
+import ch.epfl.bluebrain.nexus.delta.sdk.Lens
+import ch.epfl.bluebrain.nexus.delta.sdk.model.{Envelope, Event}
+import ch.epfl.bluebrain.nexus.sourcing.EventLog
+import monix.bio.{IO, UIO}
+
+trait EventLogSyntax {
+  implicit final def eventLogSyntax[State, E <: Event](eventLog: EventLog[Envelope[E]]): EventLogOpts[State, E] =
+    new EventLogOpts(eventLog)
+}
+
+/**
+  * Provide extension methods for an EventLog of Envelope[Event]
+  *
+  * @param eventLog the eventLog
+  */
+final class EventLogOpts[State, E <: Event](private val eventLog: EventLog[Envelope[E]]) extends AnyVal {
```

Comment:
We try to keep the syntax without any logic, just forwarding to the object, trait or class with its logic. You can see an example of that [here](https://github.com/BlueBrain/nexus/blob/master/delta/rdf/src/main/scala/ch/epfl/bluebrain/nexus/delta/rdf/syntax/JsonSyntax.scala#L67). That's why in my previous PR I said to keep the logic in `EventLogUtils` and to call it from there to here